src-java/connecteddevices-topology/connecteddevices-storm-topology/src/main/java/org/openkilda/wfm/topology/connecteddevices/service/PacketService.java

```diff
@@ -75,52 +80,70 @@ public PacketService(PersistenceManager persistenceManager) {
      * Handle LLDP info data.
      */
     public void handleLldpData(LldpInfoData data) {
-        transactionManager.doInTransaction(() -> {
-
-            FlowRelatedData flowRelatedData = findFlowRelatedData(data);
-            if (flowRelatedData == null) {
-                return;
-            }
+        FlowRelatedData flowRelatedData = findFlowRelatedData(data);
+        if (flowRelatedData == null) {
+            return;
+        }
 
-            SwitchConnectedDevice device = getOrCreateLldpDevice(data, flowRelatedData.originalVlan);
+        String uniqueIndex = createUniqueLldpIndex(data, flowRelatedData.getOriginalVlan());
```

Comment:
Would something like createUniqueLldpIndexString / createUniqueLldpIndexValue / your option be better?

Reply:
One thought is that there are methods in this class called `createMessageKey` so maybe stick to this pattern? The methods are similar. 
Otherwise, what came to my mind is `name`. It is an index name, right? Maybe also replace create with get because `create unique index` is a well-known SQL command. To me, `getUniqueLlldIndexName` and `getUniqueArpIndexName` look better

Reply:
In DB we have the property: `switch_connected_device.unique_index`. So, `UniqueIndex` is like the name of the field and the method builds the content of the field. Maybe `buildUniqueLlldIndex`, but it is again similar to the SQL command? @niksv , @pablomuri , @durgakako @IvanChupin @pkazlenka , @artHalMirantis What do you think?

Reply:
For me getSomething() means that there is certainly no create operation.
createSomething() obviously create something, but I am not sure, should I expect that this method create and store and get, or just only create and get..
I would use generateSomeIndex(), at least for me it would be clear that we certainly generating some index without storing it. But usually for me, generate is somehow related to uuid format.. idk should we use generate with not uuid format, on the other hand, we use word generate to create some public key for ssh, so maybe this word could be used..:)

Reply:
If there are no other opinions will update to `buildUniqueLlldIndex`, because from my point of view creating is too generic, and generating is like we make something from nothing but building, is like making something from smaller parts (exactly our case).
Thank you for your thoughts!

Reply:
I agree with @niksv @IvanChupin. I prefer "generate" but "build" is also ok.